cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -223,6 +223,41 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
+	zeroGracePeriod := int64(0)
+	unknownStatePodWithDeletionTimestamp := &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              "bar",
+			Namespace:         "default",
+			DeletionTimestamp: &metav1.Time{Time: time.Now().Add(-1 * PodLongTerminatingExtraThreshold)},
```

Comment:
Could you modify this test to also inject the time? I think you'll only need to change `rsPodDeleted`, shouldn't be much work.

Reply:
done

Reply:
Thank you!